examples/guestbook-go/README.md

```diff
@@ -4,7 +4,7 @@ This example shows how to build a simple multi-tier web application using Kubern
 
 ### Step Zero: Prerequisites
 
-This example assumes that you have forked the repository and [turned up a Kubernetes cluster](../../docs/getting-started-guides):
+This example assumes that you have forked the repository and [turned up a Kubernetes cluster](/docs/getting-started-guides):
```

Comment:
Since this is a user facing example (and our users wont be expected to have a fork or build anything), ill remove mention of it as part of #9398